WeakAuras/RegionTypes/RegionPrototype.lua

```diff
@@ -3,6 +3,67 @@ local L = WeakAuras.L;
 
 WeakAuras.regionPrototype = {};
 
+-- Adjusted Duration
+
+function WeakAuras.regionPrototype.AddAdjustedDurationToDefault(default)
+  default.useAdjustededMax = false;
+  default.useAdjustededMin = false;
+end
+
+function WeakAuras.regionPrototype.AddAdjustedDurationOptions(options, data, order)
```

Comment:
Is there a guideline for when to put AceConfig definitions in WeakAurasOptions and when not?